src/content/docs/kubernetes-pixie/kubernetes-integration/installation/kubernetes-integration-install-configure.mdx

```diff
@@ -1367,9 +1367,11 @@ There are three ways to install the Kubernetes integration in your GKE Autopilot
         title="Guided install (recommended)"
     >
 
-Follow the instructions in the **Guided install** tab above. 
+Follow the instructions in the **Guided install** tab above where you can choose any of these 3 options. 
 
-The guided install will ask if you are installing in a GKE Autopilot cluster. After you enter `Y`, we will set `privileged`, `controlPlane`, `logging`, and `pixie` to `false`. The install will also set the kubelet scheme to http and kubelet port to 10255. 
+* New Relic CLI
```

Comment:
This is a little confusing. My understanding is that they can install GKE autopilot using the same guided install instructions as seen in the first tab of this doc (Guided install (recommended)). That makes sense. I'm not sure how the 3 bullets fit into it. 

How do the bullet points for Helm command and manifest found in the guided collapser differ from the collapsers in this section? Should the bulleted list in the guided install collapser be broken out to the introduction? For example:

> There are three ways to install the Kubernetes integration in your GKE Autopilot cluster.
> 
> * The New Relic CLI found in our guided install
> * A Helm command with pre-populated required values
> * A plain manifest with pre-populated required values

Then the collapser titled guided install would just read:

> You can install in GKE autopilot using the same install flow as our [guided install](/docs/kubernetes-pixie/kubernetes-integration/installation/kubernetes-integration-install-configure/#guided-install) tab.